src/xact.cc

```diff
@@ -422,6 +487,7 @@ bool xact_base_t::finalize()
   return true;
 }
 
+// TODO: fix for virtual post balancing.
 bool xact_base_t::verify()
 {
```

Comment:
When I do try to fix this, how should I test it? My current guess is this code path gets called when there are auto xacts?